src/commands/zrange-command.common.js

```diff
@@ -17,3 +17,35 @@ export function slice(arr, s, e) {
   }
   return arr.slice(start, end + 1);
 }
+
+export function parseLimit(input) {
```

Comment:
Appreciate that you take the time to refactor things and improve the codebase while adding features 👏👌